### DISCUSSION TITLE
🌍 new strings for translators 🌎

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#402] [Announcements]
I'll use this thread to publish new strings to be translated, so you can subscribe and be notified.
Don't post here, create a different discussion.

**Comment by rejetto** [comment]
"upload_dd_hint": "You can upload files doing drag&drop on the files list"

**Comment by rejetto** [comment]
0.51.0
now you you can move files around, if you have to delete+upload permissions, and it uses the "cut & paste" paradigm
```
        "Cut": "Cut",
        "n_items": "{n,plural, one{# item} other{# items}}",
        "good_bad": "{good} moved, {bad} failed",
        "after_cut": "Your selection is now in the clipboard.\nGo to destination folder to paste.",
        "Cancel clipboard": "Cancel clipboard",
        "to_clipboard_source": "To folder",
        "Paste": "Paste",
        "clipboard_list": "Items in clipboard:"
```

**Comment by rejetto** [comment]
0.52.0

        "Close": "Close",
        "Folder": "Folder",
        "Web page": "Web page",
        "Link": "Link",
        "Auto-play": "Auto-play",
        "autoplay_seconds": "Seconds to wait on images",
        "Select all": "Select all",
        "go_first": "Go to first item",
        "go_last": "Go to last item",
        "Shuffle": "Shuffle",
        "Repeat": "Repeat",
        "showHelpListShortcut": "From the file list, click holding {key} to Show quickly",
        "Invalid value": "Invalid value",
        "upload_skipped": "{n} skipped",
        "Overwrite policy": "Overwrite policy",
        "Rename to avoid overwriting": "Rename to avoid overwriting",
        "Overwrite existing files": "Overwrite existing files"

**Reply by rejetto** [reply]
i forgot to add "Menu", sorry

**Reply by PH5HP** [reply]
Hi,

So that row is if I put it on top of the last additions:

"Menu": "Menu",

Is it not?... If so I will add this.

Kind regards,
Henk - PH5HP - https://fileserver.ph5hp.nl

**Reply by rejetto** [reply]
the order doesn't matter

**Reply by rejetto** [reply]
i've just found an error on "enter_comment".
it's correct english content is
        "enter_comment": "Comment for {name}",
If you can, please adjust it for your language.

**Comment by djmakers** [comment]


**Reply by BussyBakks** [reply]
hfs-lang-(2 word of ur language name).json

**Reply by BussyBakks** [reply]
example of vietnamese: hfs-lang-vi.json

**Reply by rejetto** [reply]
please refer to this page https://lgithub.com/rejetto/hfs/wiki/Translation

that said, i have to delete your message because this is a reserved  thread

**Comment by rejetto** [comment]
0.53.0

        "clipboard": "Clipboard ({content})", // pic1
        "to_clipboard_source_tooltip": "Go to the folder where the clipboard contents are located", // pic1
        "more_items": "{n} more item(s)", // pic2
        "Show details": "Show details", // pic3
        "upload_conflict": "already exists",
        "Logged in": "Logged in", // pic4
        "Logged out": "Logged out"

pic1 (files cut)
<img width="965" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/957bb66d-be81-4d12-abad-938cd9169300">

pic2 (upload)
<img width="469" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/2fd44bbf-faac-4810-bb70-13aa59a09c2a">

pic3
<img width="267" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/b3ac80b4-1ced-45ab-a268-8b6523acbf3d">

pic4 (after log in)
<img width="157" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/1aedeec6-e537-4ae7-9b24-487e029ff692">

**Comment by rejetto** [comment]
0.54.0
> "Cancel": "Cancel",
> ["allow_session_ip_change": "Allow IP change during this session"](https://github.com/user-attachments/assets/92076964-14f0-4df9-b958-28853ce2f551)

**Comment by rejetto** [comment]
0.55.0 is almost ready, time for translation!
> ["focus_hint": "By typing on your keyboard, you search and focus elements of the list.",](https://github.com/user-attachments/assets/ac4137d7-198b-477a-9ae4-c81510f772c1)
> [ "copy_links": "Copy links"](https://github.com/rejetto/hfs/discussions/803#discussioncomment-11414389)

**Comment by rejetto** [comment]
0.56.0 is coming

> ["Calculate": "Calculate",](https://github.com/user-attachments/assets/83f74840-3509-4802-ae0d-97dce406c28c)
>["Creation": "Creation",](https://github.com/user-attachments/assets/83f74840-3509-4802-ae0d-97dce406c28c)
> ["creation": "creation",](https://github.com/user-attachments/assets/bf4ad35b-0b13-436b-b303-b1fea6aa486c)
>["required_change_password": "You are required to change your password",](https://github.com/user-attachments/assets/cb563ef7-51b8-45bd-930f-4d28e2b1f99b)
>["Wildcards": "Wildcards",](https://github.com/user-attachments/assets/05a23a70-2193-4394-ab07-55a4b9a02a7e)
>"Considering resume": "Considering resume"

See linked screenshots to better understand the context.
Message "Considering resume" will be displayed in the rare case where an incomplete upload was left by HFS version before 0.56, or because HFS process is killed. In such case the browser will start examining the local file to see if it matches the partial file on the server, which can take several seconds, and it will display such message during this time, to tell the user it is working on such task. The check is necessary because the file may have changed in the meantime, and resuming a different file would result in a corrupted one.

"creation" appears in lower and upper cases but in different places.
